### DISCUSSION TITLE
Billed cost vs Effective cost

### DISCUSSION THREAD

**Discussion by saleemmy** [microsoft/finops-toolkit#718] [Q&A] [answered]
The default powerbi reports and dashboards present effective cost which is different from the billed cost. In my opinion it is the billed cost which is the same as the invoice cost to be shown and all the savings and other costs comparison to be against the billed cost.

**Comment by DirkBrinkmann** [comment, marked as answer]
BilledCost = Actual Cost = Cost on your invoice

EffectiveCost = Amortized Cost

See FOCU Spec

Effective Cost represents a cost inclusive of the impacts of all reduced rates and discounts, augmented with
the amortization of relevant purchases (one-time or recurring) paid to cover future eligible charges. The
amortized portion included should be proportional to the Pricing Quantity and the time granularity of the
data. This cost is denominated in the Billing Currency. The Effective Cost is commonly utilized to track and
analyze spending trends.
This column resolves two challenges that are faced by practitioners:
1. Practitioners need to amortize relevant purchases, such as upfront fees, over the duration of the
commitment and distribute them to the appropriate reporting groups (e.g.tags, resources).
2. Many commitment-based discount constructs include a recurring expense for the commitment for
every billing period and must distribute this cost to the resources using the commitment. This

**Reply by flanakin** [reply]
To add to this, if we calculate savings against the billed cost, you wouldn't be able to see the savings from commitment discount usage, which has 0 billed cost. 

Let us know if that helps, @saleemmy.

**Comment by saleemmy** [comment]
Sorry maybe I was not clear. when I say savings against the billed cost, I mean what is the savings achieved compared to the billed cost. Of course the savings to be calculated against the list cost. The billed cost which is the invoice is after all the discounts are applied. 

since I  dont see the correct savings, I believe the calculations are to be 
The RI savings are to be shown against the list cost. Each VM has a savings discount accordingly based on VM series, SKU, size, storage etc. and savings achieved vary across the landscape.
The Negotiated discount savings are to be shown against list cost and is a blanket discount across all resources except the RI and SP applied.

The amortized cost which is now renamed as Effective cost becomes helpful to showcase when there are upfront payments of RI or SP.

It is all different ways of showing the costs. For us as FinOps experts, we need to see the cost in various ways and calculations. But when other stakeholders like Leaders, business, finance they would like to see the actual payments monthly, quarterly, yearly along with the forecast for remaining months.  

Basically, there are three comparisons, budget vs actual spend vs forecast

**Reply by flanakin** [reply]
The challenge with that is that commitment discount purchases aren't fully realized yet. Let's say you purchase a reservation on Jan 1 for $365 which gives you 50% off your usage costs ($1/day contracted and $2/day list). After Jan 10, BilledCost = $365 and ListCost = $20. You won't be able to calculate savings from that reservation until the end of the commitment term or at least the break-even point.

(Actually, ListCost = $750 after Jan 10 because the ListCost of the reservation purchase would be $730 plus the $20 in usage cost after 10 days. This is especially bad because ListCost and ContractedCost double-count commitment discount purchases, but that needs to be accounted for regardless.)
